src/track/beatmap.cpp

```diff
@@ -752,3 +768,84 @@ double BeatMap::calculateBpm(const Beat& startBeat, const Beat& stopBeat) const
 
     return BeatUtils::calculateBpm(beatvect, m_iSampleRate, 0, 9999);
 }
+
+mixxx::Signature BeatMap::getSignature(double dSample) const {
+    QMutexLocker locker(&m_mutex);
+    if (!isValid())
+        return mixxx::Signature(0,0);
+
+    auto result = mixxx::Signature(4,4);
```

Comment:
The default signature should be a named constant

Reply:
Good point.
Done